vitest-setup.ts

```diff
@@ -25,6 +25,12 @@ const BroadcastChannelMock = vi.fn(() => ({
 
 vi.stubGlobal('BroadcastChannel', BroadcastChannelMock);
 
+const cryptoMock = {
+  randomUUID: () => 'test-uuid-' + Math.random().toString(36).substring(2, 9),
+};
+
+vi.stubGlobal('crypto', cryptoMock);
```

Comment:
Is there a reason you added this mock? Looks like it's currently not being used anywhere.

Reply:
I added it to make the toaster test work. It kept failing. It looks like it's not used if you search for the import, but from what I understand this file runs before everything else. Adding this fixed the Toaster test.